nilearn/plotting/displays.py

```diff
@@ -746,12 +746,9 @@ def add_edges(self, img, color='r'):
 
         Parameters
         ----------
-        map: 3D ndarray
-            The 3D map to be plotted. If it is a masked array, only
-            the non-masked part will be plotted.
-        affine: 4x4 ndarray
-            The affine matrix giving the transformation from voxel
-            indices to world space.
+        img: Niimg-like object
+            See http://nilearn.github.io/manipulating_images/input_output.html.
+            Provides image to plot.
```

Comment:
May be we should adapt well to the same conventions as was before. You now know better what to write following from issue. :)

"The 3D map to be plotted. If it is a masked array, only the non-masked part will be plotted"